Library/Homebrew/cask/audit.rb

```diff
@@ -572,10 +572,15 @@ def check_signing
         tmpdir = Pathname(tmpdir)
         primary_container.extract_nestedly(to: tmpdir, basename: downloaded_path.basename, verbose: false)
         artifacts.each do |artifact|
-          result = system_command("codesign", args: [
-            "--verify",
-            tmpdir/artifact.source.basename,
-          ], print_stderr: false)
+          path = case artifact
+          when Artifact::Moved
+            tmpdir/artifact.source.basename
```

Comment:
Does this work?

```suggestion
            artifact.target
```

I don't think there's a reason we need to inspect the downloaded files instead of the installed files, right? Especially since `Artifact::App` just copies files directly (unless I'm forgetting something) from `#source` to `#target`.

This would allow the `tmpdir` and `primary_container.extract_nestedly(to: tmpdir, basename: downloaded_path.basename, verbose: false)` to be removed I'm pretty sure.

Reply:
Actually, nothing should be downloaded in this check at all. `brew audit --online` downloads everything at the start, so having `downloaded_path = download.fetch` here just re-downloads everything (in reality it's cached so no big deal but it is unnecessary).